src/lib.rs

```diff
@@ -23,17 +23,20 @@ use std::panic::{catch_unwind, AssertUnwindSafe};
 
 use cosmwasm_vm::{
     call_handle_raw, call_init_raw, call_migrate_raw, call_query_raw, Backend, Cache, Checksum,
-    InstanceOptions,
+    InstanceOptions, Size,
 };
 
 use crate::args::{CACHE_ARG, CODE_ID_ARG, ENV_ARG, GAS_USED_ARG, INFO_ARG, MSG_ARG, WASM_ARG};
 use crate::cache::{cache_t, to_cache};
 use crate::error::{handle_c_error, Error};
 
-fn into_backend(db: DB, api: GoApi, querier: GoQuerier) -> Backend<GoStorage, GoApi, GoQuerier> {
+// TODO: Remove constant and make configurable by user
```

Comment:
Yup. Let's get this as a new arg. Otherwise :100:

Reply:
Done in bbeb7a993442efeddd00adb62fa21d338972cd96